_field-types/mapping-parameters/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Mapping parameters
+nav_order: 75
+has_children: true
+has_toc: false
+---
+
+# Mapping parameters
+
+Mapping parameters are used to configure the behavior of fields in an index. For parameter use cases, see the mapping parameter's respective page.
+
+The following table lists OpenSearch mapping parameters.
+
+Parameter | Description
+:--- | :---
```

Comment:
Line 18: I'm not following "Specifies a field-level query time to boost." Can we rephrase for clarity? Line 21: It looks like we provide two sets of allowed values.

Reply:
Revised as follows: Specifies a field-level boost factor applied at query time. Allows you to increase or decrease the relevance score of a specific field during search queries.